### PR TITLE
LPS-168228 | Test Automation | Master

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-168228

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase [hunk 1]
```diff
@@ -117,6 +117,78 @@ definition {
 		}
 	}
 
+	@description = "LPS-168228 - Verify it's possible to add many custom fields in the custom layout"
+	@priority = "4"
+	test CanAddManyFieldsOnLayout {
+		task ("Given an object with 8 fields is created") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase:123]: Update this task block to reflect your actual number of fields

Comment (JhossephG) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase]: Done!

**Comment by epark18** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:forward

**Pull request closed without merging**